Palindrome/palindrome_no.py

```diff
@@ -0,0 +1,34 @@
+# Python program to find whether a number is a palindrome or not
+
+# Function to reverse a number
+def reverse(num):
+    # Initialize variable
+    rev = 0
+
+    while ( num > 0 ) :
+        dig = num % 10
+        rev = rev * 10 + dig
+        num = num // 10
+    # Returning reversed number
+    return rev    
+    
+# --- main --- 
+num = int(input("Enter a number:"))
+a  = reverse(num)
+
+#Comparing the reversed number with original number
```

Comment:
Add a space after #

Reply:
done with the changes @somya-kapoor  maam